lib/travis/build/addons/jwt.rb

```diff
@@ -0,0 +1,43 @@
+require 'travis/build/addons/base'
+require 'jwt'
+
+module Travis
+  module Build
+    class Addons
+      class Jwt < Base
+        SUPER_USER_SAFE = true
+
+        def before_before_script
+          secrets = nil
+          if config.is_a?(String)
+            secrets = [config]
+          elsif config.is_a?(Array)
+            secrets = config
+          else
+            secrets = config.values
```

Comment:
Awesome.

Next break I'll update the code, tests, and docs

On Jul 26, 2016 9:21 AM, "Hiro Asari" <notifications@github.com> wrote:

> In lib/travis/build/addons/jwt.rb
> <https://github.com/travis-ci/travis-build/pull/356#discussion_r72284503>:
>
> > +require 'jwt'
> > +
> > +module Travis
> > +  module Build
> > +    class Addons
> > +      class Jwt < Base
> > +        SUPER_USER_SAFE = true
> > +
> > +        def before_before_script
> > +          secrets = nil
> > +          if config.is_a?(String)
> > +            secrets = [config]
> > +          elsif config.is_a?(Array)
> > +            secrets = config
> > +          else
> > +            secrets = config.values
>
> I don't think singling out strings make sense here.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/travis-ci/travis-build/pull/356/files/b5a1fb6de21ee237ca552a761d612899bf11ee0f#r72284503>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGuB3uuKdpxht0CgyCRY7l1XjgBRJM3ks5qZjQRgaJpZM4DHeYE>
> .
>